messaging-modules/spring-apache-camel/.gitignore

```diff
@@ -1 +1,2 @@
-/src/test/destination-folder/*
\ No newline at end of file
+/src/test/destination-folder/*
+/output/
```

Comment:
Because some of the class that was migrated from spring-boot-camel output the some changes in the output folder and we want to ignore that files that are generated during running the classes.